lib/cangallo/qcow2.rb

```diff
@@ -160,10 +160,12 @@ def self.create_from_base(origin, destination, size=nil)
 
     def self.create(image, parent=nil, size=nil)
       cmd = [:create, '-f qcow2']
-      cmd << "-o backing_file=#{parent}" if parent
+      if parent
+        backing_fmt = File.extname(File.basename(parent)).delete_prefix('.')
```

Comment:
It could be better to inspect the backing image and get its format. I'll take a look to check if this can be done easily. For now this fixes the problem at least with the tests.